### DISCUSSION TITLE
Email-Table display a table field as a hyperlink

### DISCUSSION THREAD

**Discussion by SoykaObecna** [EvotecIT/PSWriteHTML#243] [Q&A]
Hello,

I have an array that is a datasource for Emai-Table. One property of the array contains a text. It is a link to a file (file://server\share\fileN.xlsx). Please, is it possible to render this column as a hyperlink in an email using Email-Table?

Thanks to the author for this wonderfull module. Thanks to everyone that can point me in the right direction.

Jiri

**Comment by PrzemyslawKlys** [comment]
You need to modify the source of your DataSource and modify the value to generate New-HTMLLink with file://server\share\fileN.xlsx inside the DataSource, and once this is done make sure to use EmailTable -InvokeHTMLTags this will trigger conversion of any HTML code to be "executed".
